projects/openyurt/2024-OpenYurt-incubation.md

```diff
@@ -0,0 +1,387 @@
+# OpenYurt Incubation Due Diligence
+
+- Link to [Incubation application issue](https://github.com/cncf/toc/pull/797)
+
+<!-- This template provides the TOC with the outline for completing due diligence of a project to move levels. This universal template is designed to capture all criteria so the TOC may ensure prior level criteria do not regress. As part of completing the due diligence, the TOC member should update the template to convey the level the project applied for the criteria by bolding the level indicated where the criteria is relevant. -->
+
+## Incubation Evaluation Summary for OpenYurt
+
+### Criteria Evaluation
+
+[Ricardo Rocha](https://github.com/rochaporto) conducted both due diligence and adopter interviews for Incubation. The project has completed the criteria that shows its maturity at the applied level.
+
+The TOC would like to highlight some areas from the project review:
+* OpenYurt has an active and diverse community, and adopters have highlighted this as one of its main strenghts
+* Its architecture extends instead of replacing existing core components in Kubernetes. This is an approach that differentiates it from other solutions in the ecosystem and that was also highlighted as a key strength by adopters
+* The governance and contributors guides are well defined but were last updated over 3 years ago. The TOC suggests a review together with TAG Contributor Strategy to consider if improvements can be made to the current member roles and contributor ladder
+
+The following required incubation items were raised and implemented by the project during the review period:
+* A security self assessment following the template from TAG-Security
+* Setting up of the best practices badge from the OpenSSF
+
+Other items suggested for incubation were taken as issues in the project and have also been followed up:
+* Documenting integrations with other CNCF and non-CNCF projects to facilitate adoption by end users
+* Documenting the process for definition of the project roadmap
+* Clarification of public contact for security vulnerability reports
+
+### Adoption Evaluation
+
+The adopter interviews showed project usage at a level compatible with CNCF incubation. Adopters integrate OpenYurt for management of edge devices on a variety of use cases, and highlight its extensibility of upstream vanilla Kubernetes as one of its key strengths - some adopters highlighted this is a differentiating point when compared to similar projects in the ecosystem which do the same by replacing some of the core components in Kubernetes. The project has a well established and responsive community in a variety of communication channels.
+
+Adopters also suggested improvements in resource consumption, multi-tenancy support and recommend the project focuses on edge AI use cases looking forward.
```

Comment:
Clarification: do the adopters recommend only focusing on edge AI use cases looking forward? or in addition to current edge use cases? does the project not include any edge AI use cases at this time?

Reply:
The adopter interview summaries are great. Answering my own comment above from Adopter Interview 4:

> It also recommends focusing on further integration with AI tools in particular EdgeAI modules for inference.

Reply:
Thanks @angellk . Indeed that was the case - extending the focus, inference was highlighted.